README.md

```diff
@@ -24,5 +24,11 @@ spring:
 ```
 
 7. Set application run configuration to `local` using the following flag: `spring.profiles.active=local`. 
+8. Install a recent version of Node.js from [here](https://nodejs.org/en/download/).
+9. Front-end local build: 
+```
+cd book-exchange-website
```

Comment:
please rename this to `alexandria-ui` instead of `book-exchange-website`

Reply:
I renamed it to `alexandria-ui`.